src/protocols/ssh/ssh.c

```diff
@@ -296,9 +301,15 @@ void* ssh_client_thread(void* data) {
 
     }
 
+    /* Set up the ttymode array prior to requesting the PTY */
+    if (guac_ssh_ttymodes_init(ssh_ttymodes, sizeof(ssh_ttymodes),
+            num_tty_opcodes, (guac_ssh_ttymode){ GUAC_SSH_TTY_OP_VERASE, settings->backspace}))
+        guac_client_abort(client, GUAC_PROTOCOL_STATUS_SERVER_ERROR, "Error configuring TTY mode encoding.");
```

Comment:
What do you mean by "configuring TTY mode encoding"? Is there a way to phrase this which would better represent what is failing?

Reply:
Tried to clean up this error message bit.